src/lib/viewers/doc/BoxExcel/index.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
+import Workbook from './workbook';
+
+class BoxExcel {
```

Comment:
This should probably be BoxExcel.js when it gets moved into `src/lib/viewers/sheet/BoxExcel.js`